src/Core/Material/BlinnPhongMaterialModel.cpp

```diff
@@ -24,6 +26,81 @@ void BlinnPhongMaterialModel::displayInfo() const {
     print( hasNormalTexture(), " Normal Texture : ", m_texNormal );
     print( hasOpacityTexture(), " Alpha Texture  : ", m_texOpacity );
 }
+Utils::Color Material::BlinnPhongMaterialModel::evalBSDF( Vector3f w_i,
```

Comment:
```suggestion
Utils::Color Material::BlinnPhongMaterialModel::evalBSDF( Vector3 w_i,
```
Do not use `Vector3f` to make your code compatible with the numeric format (float or double) of the Radium engine.
By using `Vector3`, you are sure components are of type `Scalar`.
Constant then should be written as `1.414_ra`